.github/workflows/build-all-matrix.yaml

```diff
@@ -0,0 +1,188 @@
+name: All Kernels and Hooks
+on:
+  #schedule:
+  #  # every day at 5am UTC
+  #  - cron: '0 5 * * *'
+  workflow_dispatch:
+
+env: # Global environment, passed to all jobs & all steps
+  # Unfortunately quay.io seems to go down a lot. Allow to use ghcr.io easily:
+  REGISTRY: "quay.io" # or quay.io, determines which will be logged-in to
+  #HOOK_KERNEL_OCI_BASE: "ghcr.io/${{ github.repository_owner }}/tinkerbell/kernel-"
+  #HOOK_LK_CONTAINERS_OCI_BASE: "ghcr.io/${{ github.repository_owner }}/tinkerbell/linuxkit-"
+  
+  # Apart from the quay/ghcr coordinates above (used for both pulling & pushing), we might also want to
+  # log in to DockerHub (with a read-only token) so we aren't hit by rate limits when pulling the linuxkit pkgs.
+  # To do so, set the secret DOCKER_USERNAME and DOCKER_PASSWORD in the repo secrets, and set the below to yes.
+  LOGIN_TO_DOCKERHUB: "${{ github.repository_owner == 'rpardini' && 'yes' || 'no' }}"
```

Comment:
Sorry, DockerHub isn't used in Tinkerbell. We'll need to remove this.

Reply:
Definitely. I just wanted to exemplify the conditional -- not everyone will have it I guess. 
Since I added LK caching (to GHA cache), the need for this is lower as well.

Reply:
Removed conditional for this PR.